Jenkinsfile.groovy

```diff
@@ -163,11 +163,16 @@ pipeline {
             
             sh "bash ./build/deploy-demo.sh ${env.BRANCH_NAME}"
```

Comment:
Maybe out of scope: I had a look at this script in my PR and it could easily be removed by just executing the commands here directly

Reply:
I would consider this out of scope. I'm not sure what is best (having a dedicated script for that or having it inline here)